claim/README.md

```diff
@@ -41,44 +38,67 @@ There are two important notes regarding connecting nodes:
 
 ## How to connect a node
 
+There will be two main flows from where you might want to connect a node to Netdata Cloud.
```

Comment:
There is also the option of adding nodes from table view (nodes tab), where a user can press the (+) button. Should we mention this as well?

Reply:
I see what you mean it opens the "Manage War Room / Add Nodes" tab on the right... that one skipped me 😓
For consistency I believe it is best to include it, will add this